### ISSUE TITLE
Unable to view any databases in Object explorer

### ISSUE THREAD

**Issue by Ignobilis123** [microsoft/azuredatastudio#25848]
Type: <b>Bug</b>

Object Exporer task didn't complete within 45 seconds

Azure Data Studio version: azuredatastudio 1.49.1 (2f696aa95a13ccabd7e0a4431d1d759e3a7b8cfd, 2024-08-13T18:12:38.831Z)
OS version: Windows_NT x64 10.0.19045
Restricted Mode: No
Preview Features: Enabled
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-1265U (12 x 2688)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.44GB (8.87GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
azuremonitor|Mic|0.1.9

</details>

**Comment by kburtram** [comment]
@Ignobilis123 this error means that the database did not complete the authentication & metadata queries to populate object explorer within the 45 second timeout.  This can occur for several reasons, such as slow connection to server, resuming a sleeping serverless instance, etc.  

Does this error occur consistently when retrying the connection?  Does the server work fine with other tools, like SSMS or sqlcmd?

**Comment by Ignobilis123** [comment]
yes its constant. SSMS, SQL CMD and Python in VSCode all good. My first time using Data Studio and not a great impression so far :|
